src/taxes/components/TaxPageTitle/TaxPageTitle.tsx

```diff
@@ -0,0 +1,28 @@
+import HorizontalSpacer from "@saleor/apps/components/HorizontalSpacer";
+import PreviewPill from "@saleor/components/PreviewPill";
+import { sectionNames } from "@saleor/intl";
+import { makeStyles } from "@saleor/macaw-ui";
+import React from "react";
+import { FormattedMessage } from "react-intl";
+
+const useStyles = makeStyles(
+  () => ({
+    wrapper: {
+      display: "flex",
+    },
+  }),
+  { name: "TaxPageTitle" },
+);
+
+export const TaxPageTitle: React.FC = () => {
```

Comment:
React.FC is not good practice so far especial when you don't use children
```suggestion
export const TaxPageTitle: = () => {
```